src/astro/sql/operators/append.py

```diff
@@ -9,8 +11,6 @@
 if TYPE_CHECKING:
     from airflow.models.xcom_arg import XComArg
```

Comment:
This `if TYPE_CHECKING` guard seems unnecessary?

Reply:
Fixed